setup.py

```diff
@@ -5,26 +5,25 @@
 
 
 setup(
-    name="change_this_to_name_of_package",
-    version="0.1",
-    description="CREATE A DESCRIPTION",
-    url="https://github.com/CuriBio/CHANGE_THIS_TO_NAME_OF_REPO",
+    name="misc_test_utils",
+    version="0.2",
+    description="Persisting objects.",
```

Comment:
description="Utility functions for running unit and integration tests. With a focus on domain models and MySQL databases."